diff --git a/node_modules/react-native-quick-crypto/react-native-quick-crypto.podspec b/node_modules/react-native-quick-crypto/react-native-quick-crypto.podspec
index 31a98d9..b880870 100644
--- a/node_modules/react-native-quick-crypto/react-native-quick-crypto.podspec
+++ b/node_modules/react-native-quick-crypto/react-native-quick-crypto.podspec
@@ -41,4 +41,5 @@ Pod::Spec.new do |s|
   s.dependency "React-Core"
   s.dependency "React"
   s.dependency "React-callinvoker"
+  s.dependency "ReactCommon"
 end
